searx/engines/xpath.py

```diff
@@ -72,6 +72,7 @@
 from lxml import html
 from searx.utils import extract_text, extract_url, eval_xpath, eval_xpath_list
 from searx.network import raise_for_httperror
+from searx import logger
```

Comment:
No need to import logger.  In the engine modules, logger is already added by some monkey patching of the SearXNG core / compare https://github.com/searxng/searxng/commit/73a6da4dd9ab95555969d42aa64bbc1c78bb0a71

Reply:
> No need to import logger. In the engine modules ...

OK, now I understand why you have been added the logger to the xpath.py module .. today I had this (randomly) error in the unit tests .. its fixed in https://github.com/searxng/searxng/pull/3609